addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13315,6 +13315,7 @@ msgstr ""
 
 #: xbmc/video/GUIViewStateVideo.cpp
 #: xbmc/playlists/SmartPlayList.cpp
+#: There is option to sort list also by Original Title (ListItem.OriginalTitle)
```

Comment:
nope. the line must start with #. (hashmark, followed by dot)
And I'm honestly not sure this description is actually helpful. You should at least explain what an "original title" is, so a translator gets an idea *what* this string denotes.

Reply:
The term "original title" has been there for a long time and this PR just cleans up the usage comments. I don't see why he has to add a detailed description of what "original title" means.

Reply:
Translators in Weblate may not be familar with our code to look up the details there. "Original title" is quite a generic term - so, a translator might ask: original title of what? Of a movie? A PVR recording? That might make a difference in non-english languages. A comment explaining what this string is actually used for is really very helpful - says somebody who translated many hundreds of strings for Kodi, skins and add-ons.

Reply:
#. Original title of the movie/tvshow 

is fine for you?

I am struggling with better definition of original, as this term is known for me since years (original title vs "country distributor title" which could differ or not )..
![image](https://user-images.githubusercontent.com/7437057/148634724-3459d597-92cd-4a74-9d86-64cf48ee31ae.png)